src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -1,147 +1,166 @@
-        }
-        HashMap<String, Object> chromePrefs = new HashMap<String, Object>();
-        chromePrefs.put("profile.default_content_settings.popups", 0);
-        chromePrefs.put("download.default_directory", downloadPath);
-        /**
-         * Save Chrome Options
-         */
-        ChromeOptions options = new ChromeOptions();
-        HashMap<String, Object> chromeOptionsMap = new HashMap<String, Object>();
-        options.setExperimentalOption("prefs", chromePrefs);
-        chromePrefs.put("safebrowsing.enabled", "true");
-        options.addArguments("--safebrowsing-disable-download-protection");
-        options.addArguments("disable-popup-blocking");
-
-        DesiredCapabilities cap = DesiredCapabilities.chrome();
-        cap.setCapability(ChromeOptions.CAPABILITY, chromeOptionsMap);
-        cap.setCapability(CapabilityType.ACCEPT_SSL_CERTS, true);
-        cap.setCapability(ChromeOptions.CAPABILITY, options);
-
-        screenshotPath = System.getProperty("user.dir") + File.separator + "test-output";
-        driver = new EventFiringWebDriver(new ChromeDriver(cap));
-        driver.manage().timeouts().implicitlyWait(DELAY, TimeUnit.SECONDS);
-        WebDriverEventListener errorListener = new AbstractWebDriverEventListener() {
-            @Override
-            public void onException(final Throwable throwable, final WebDriver activeDriver) {
-                try {
-                    takeScreenshot();
-                } catch (Exception e) {
-                    e.printStackTrace();
-                }
-            }
-        };
-        driver.register(errorListener);
-    }
-
-    /**
-     * Close browser windows and terminate WebDriver session.
-     */
-    @After("~@RegressionAPI")
-    public void afterMethod() {
-        driver.quit();
-    }
-
-    public static EventFiringWebDriver getDriver() {
-        return driver;
-    }
-
-    public static File getDownloadDirectory() {
-        return dir;
-    }
-
-    /**
-     * Take a screenshot.
-     * @param hash a string that will be inserted into the filename to avoid overwriting images.
-     * All "non-word characters" will be replaced with "_"
-     * @throws Exception if there is an exception
-     */
-    public static void takeScreenshot(final String hash) throws Exception {
-        File scrFile = ((TakesScreenshot) getDriver()).getScreenshotAs(OutputType.FILE);
-        File outFile = new File(screenshotPath + File.separator + "failed-test-" + hash.replaceAll("\\W+", "_") + ".png");
-        FileUtils.copyFile(scrFile, outFile);
-    }
-
-    /**
-     * Take a screenshot. Generates hash based on current time.
-     * @throws Exception if screenshot cannot be created
-     */
-    public static void takeScreenshot() throws Exception {
-        String hash = String.valueOf(new Date().getTime());
-        takeScreenshot(hash);
-    }
-    
-
-}
+		 */
+
+		if(System.getenv("browser")!=null && !System.getenv("browser").isEmpty()){
```

Comment:
There are a lot of code style issues that checkstyle should be warning about. I see a bunch of tabs while the team has decided spaces is what we're using. Also, there should be a space after the `if` and before the `(`, as well as around `!=` and other operators